site.yml

```diff
@@ -157,6 +157,14 @@
       tags: ['ceph', 'ceph-osd']
   environment: "{{ env_vars|default({}) }}"
 
+- name: ceph update
+  hosts: ceph_monitors:ceph_osds:controller:compute
```

Comment:
I don't get it. The tasks in the `ceph-update` role just restart ceph-mons, run some ceph calls on specific nodes (first time through or last time through), and restart ceph-osds.

I don't see any `notify` keys that would trigger handlers. I don't see any template calls.

I'm not following the logic here.